x/sp/client/cli/tx.go

```diff
@@ -715,3 +717,51 @@ Examples:
 	cmd.Flags().Int64(FlagDuration, 0, "maintenance duration requested by a SP")
 	return cmd
 }
+
+func CmdUpdateStorageProviderStoragePrice() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "update-price [sp-address] [read-price] [store-price] [free-read-quota]",
+		Short: "Update prices and free read quota of a storage provider, all prices in BNB wei",
+		Long: strings.TrimSpace(
+			fmt.Sprintf(`update the storage provider read, store price and free read quota, if there is no change to a specific value, the current value should also be provided.
+Examples:
```

Comment:
I think we should describe more detail here, how to compute read/storage price, for example, if the expected storage price is 0.023$/GB/Month, then the read-price here should be 0.023*0.58/BNB Price*1e18/1024/1024/1024/30/24/60/60

Reply:
The free-read-quota unit is bytes, for 1GB free quota, it should be 1073741824.

Reply:
added